model/variable.h

```diff
@@ -129,7 +130,21 @@ namespace minsky
       if (auto v=vValue()) return v->hypercube().dims();
       else return {};
     }
-      
+
+    std::vector<std::string> dimLabels() const {
+      std::vector<std::string> tmpLabels;	
```

Comment:
Decl should be moved to just before line 137

Reply:
Thanks, done.